src/controller/AppHandle.h

```diff
@@ -197,10 +200,13 @@ class AppHandle : private juce::Timer
     void updateOutput();
 
     //==============================================================================
-    AppConfig config;                   ///< App configuration.
-    juce::ChildProcess process;         ///< Actual app process.
-    AppState state = AppState::init;    ///< Current app state.
-    juce::String output;                ///< Output captured from the app process.
+    AppConfig config;                            ///< App configuration.
+    juce::ChildProcess process;                  ///< Actual app process.
+    AppState state = AppState::init;             ///< Current app state.
+    juce::String lastOutput;                     ///< Last available output captured from the app.
+    juce::MemoryOutputStream output;             ///< Output currently being captured.
+    juce::CriticalSection outputLock;            ///< Lock for concurrent access to output.
+    std::unique_ptr<std::jthread> captureThread; ///< Thread for asynchronous output capturing.
```

Comment:
@JanosGit Any idea why this fails building in actions CI?
I raised the OSX deployment target to 11.0 because I got the error locally (`std::jthread` only available from 11.0), then there was no problem anymore.
However, I witnessed some very strange effects on the CI when having rebased branches of open PRs: In the build logs, there were still references to code parts that were actually not part of the code anymore after rebasing.
So probably I'd give it a try and merge and see whether the error persists.

Reply:
Aha, yes, that might be the reason…
I have Apple clang 1700.3.19.1 (from Xcode 26.0.1) which supports std::jthread, and 1700.0.13.5 from Xcode 16.4 does not…
In this case I'll try to switch to the `macos-26` actions runner (public preview).